openapi.yaml

```diff
@@ -1,190 +1,513 @@
-  authorization:
-    name: Authorization
-    in: header
+  ownerIdPath:
+    in: path
+    name: ownerId
+    description: ID of owner
     type: string
     required: true
-    description: '"Bearer [token]" where [token] is your OAuth2 access token'
-  pageNumber:
-    name: page[number]
-    in: query
-    type: integer
-    required: false
-    description: Page number of results
-    default: 1
-    minimum: 1
-  pageSize:
-    name: page[size]
-    in: query
-    type: integer
-    required: false
-    description: Number of results to return
-    default: 25
-    maximum: 500
-    minimum: 1
+  parkIdPath:
+    name: parkId
+    in: path
+    description: Unique Park ID
+    required: true
+    type: string
+  parkPostBody:
+    name: parkBody
+    in: body
+    description: A park body object
+    required: true
+    schema:
+      $ref: '#/definitions/PostParkModel'
+  parkPatchBody:
+    name: parkBody
+    in: body
+    description: A park body object
+    required: true
+    schema:
+      $ref: '#/definitions/PatchParkModel'
+  ownerPostBody:
+    name: ownerPostBody
+    in: body
+    description: An owner body object
+    required: true
+    schema:
+      $ref: '#/definitions/PostOwnerModel'
+  ownerPatchBody:
+    name: ownerPatchBody
+    in: body
+    description: An owner body object
+    required: true
+    schema:
+      $ref: '#/definitions/PatchOwnerModel'
 definitions:
-  Meta:
-    properties:
-      totalResults:
-        type: integer
-        description: Total number of results
-        example: 10
-      totalPages:
-        type: integer
-        description: Total number of pages
-        example: 10
-      currentPageNumber:
-        type: integer
-        description: Page number of the returned results
-        example: 1
-      currentPageSize:
-        type: integer
-        description: Number of results per page
-        example: 25
   SelfLink:
     properties:
       self:
         type: string
+        example: request_url_here
         format: url
         description: Self-link of current resource
-  PaginationLinks:
+  OwnerResource:
+    required: [id, type, attributes]
     properties:
-      first:
-        type: string
-        format: url
-        description: The first page of data
-      last:
+      id:
         type: string
-        format: url
-        description: The last page of data
-      prev:
+        example: '0001'
+        description: A unique ID of a park owner
+      type:
         type: string
-        format: url
-        description: The previous page of data
-      next:
+        description: type of object, should be owner
+        example: owner
+      attributes: &ownerAttributes
+        required: [ownerName]
+        properties:
+          ownerName:
+            type: string
+            description: Name of park owner / organization
+            example: City of Corvallis
+  PostOwnerModel:
+    required: [type, attributes]
+    properties:
+      type:
```

Comment:
Since this `type` is the same as in the owner resource, could you reference it here?